### PR TITLE
change wrong letter

### PR DESCRIPTION
**Pull request by leonzone**

### CODE REVIEW

**Comment by leonzone** [comment]
you can see see

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# hello-Python
```
Comment (reisexu) [inline README.md:1]: just this ?

Comment (reisexu) [inline README.md:1]: I like little

**Review by reisexu — COMMENTED** [review]
change again

File: README.md (hunk 1 above)
Comment (leonzone) [reply README.md:1]: yes

Comment (leonzone) [reply README.md:1]: emmmm

**Comment by reisexu** [comment]
no!!!

**Pull request closed without merging**